ansible/roles/nginx/templates/rogeros_auth_proxy.conf.j2

```diff
@@ -6,37 +6,34 @@ upstream chronos {
         server localhost:4400;
 }
 
+{% if env_name != 'local' %}
 server {
       listen 80 default_server;
       server_name _ ;
-{% if env_name == 'local' %}
-      rewrite ^ https://$host$request_uri redirect;
-{% else %}
       return 444;
-{% endif %}
 }
 
-{% if cert_subdomain is defined %}
 server {
        listen 80;
        server_name ~^mesos.+$ ;
        location / {
              proxy_pass {{ mesos_elected_leader_url }};
        }
-}
+}{% endif %}
 
 server {
-       listen 80;
-       server_name ~^marathon.+${% if env_name == 'local' %} default_server{% endif %} ;
-       rewrite ^ https://marathon-{{env_name}}-roger.{{cert_subdomain}}$request_uri redirect;
+       listen 80{% if env_name == 'local' %} default_server{% endif %} ;
+       server_name ~^marathon.+$ ;
+       rewrite ^ https://{% if env_name == 'local' %}{{ roger_http_url_base }}{% else %}marathon-{{env_name}}-roger.{{cert_subdomain}}{% endif %}$request_uri redirect;
```

Comment:
Does this level of complexity warrant constructing a variable elsewhere that can keep the template logic cleaner?